+Base/GetClasses.m

```diff
@@ -36,11 +35,8 @@
 
     function addto(class_str)
         % Ignore classes that have filenames ending with "_invisible"
-        % Ignore classes that have an Constant invisible property set to true
         visible = ~Base.EndsWith(class_str,'_invisible');
-        try %#ok<TRYNC>
-            visible = and(visible,~logical(eval(sprintf('%s%s.invisible',prefix,class_str))));
-        end
```

Comment:
Feature NotImplemented Re: Michael.

Reply:
do we want to implement or should we scrap? implementing would require grabbing the property meta class and seeing if it was defined in this class.

Reply:
Scrap.